docs/site/tutorials/core/1-introduction.md

```diff
@@ -81,3 +81,7 @@ There are dependencies between such artifacts, such as:
 
 Some of the artifacts can be configurable, either statically at startup time or
 dynamically when the application is running.
+
+Previous: [Index](./index.md)
```

Comment:
Can we add a horizontal bar using `* * *`?

Reply:
@raymondfeng, added separators for all pages.